src/Radio/RadioGroup.js

```diff
@@ -74,6 +82,7 @@ export default class RadioGroup extends Component {
 
   state = {
     currentTabIndex: undefined,
+    focused: false, // used for optional label required/error
```

Comment:
I appreciate the comment as it's a tradeoff to keep this state up to date even if we might not need it.
E.g. Users that don't need error handling.